team/Suman-Shekhar.yaml

```diff
@@ -0,0 +1,26 @@
+# Replace CAPITALIZED placeholders with your info
+- title: Suman Shekhar
+  avatar: https://avatars.githubusercontent.com/Sumanshekhar17
+  role: Graduate Research Assistant
+  organizations:
+    - name: Rutgers University
+  bio: >
+    I am Graduate student in Ocean Modeling lab at Rutgers University, New Jersey. I study about physics of ocean warming in Mid Atlantic bight and data driven approaches in ocean modeling.
```

Comment:
What does "bight" mean? I'm new to this term and like learning new things!

Reply:
So if you see the map from Cape Hatteras and Cape Cod, it looks like someone has bitten the the US map, and that's why it's called Bight in the Middle of Atlantic Ocean! (Mid Atlantic Bight)